README.md

```diff
@@ -62,6 +62,24 @@ When using the SDK and ChatJS, you may remove the SDK from ChatJS to ensure lack
 
 `npm install amazon-connect-chatjs`
 
+#### Using ChatJS from CDN Link
+
+`amazon-connect-chat.js` bundle file is also available over a CDN.
+
+```html
```

Comment:
Is there a way Customers can always use @latest ?

Reply:
This will always default to latest. If they want older version, they can specify